tests/typ/bugs/3917-infinite-stack-spacing.typ

```diff
@@ -0,0 +1,18 @@
+// Stacks with infinite spacing were causing a panic.
```

Comment:
Due to https://github.com/typst/typst/pull/3922, this will need to be moved to the bottom of `tests/suite/layout/stack.typ` and the individual tests need to be named.

Reply:
Should be done now.